Code/RDBoost/Wrap/RDBase.cpp

```diff
@@ -48,33 +194,41 @@ void AttachFileToLog(std::string spec, std::string filename, int delay = 100) {
 #endif
 #endif
 }
+
+
+void LogDebugMsg(const std::string &msg) {
+  NOGIL nogil;
+  BOOST_LOG(rdDebugLog) << msg << std::endl;
+}
```

Comment:
These are still releasing the GIL for now, 'cause it's the old behavior, but the tests run fine without this as well.

Reply:
The only reason they released the GIL was because C++ logging doesn't need it.